problem_builder/tests/unit/test_problem_builder.py

```diff
@@ -236,8 +236,16 @@ def test_get_option(self):
             self.assertEqual(option, None)
 
     def test_student_view_calls_get_option(self):
+        random_key, random_value = random(), random()
         self.block.get_xblock_settings = Mock(return_value={})
         with patch.object(self.block, 'get_option') as patched_get_option:
+            self.runtime_mock.service.return_value = {
+                random_key: random_value,
+                random_key: random_value,
+                random_key: random_value,
+                random_key: random_value,
+                random_key: random_value,
+            }
```

Comment:
Why repeat this 5 times it has to affect. Please clean up.

Reply:
because it is expceting 5 key value pairs. that's why added it 5 times.

Reply:
Let me test if it passes the tests with single parameter. If that happens, I will keep only one.

Reply:
@umarmughal824 I believe the tests will fail because of @xitij2000 's comment. 
But you can use something like this for generating a random dict: 
```
{random.random(): random.random() for k in range(5)}
```